canonical/ubuntu.com
# Show line breaks on Certified Hardware pages

## Done

- On "Notes", show line breaks where appropriate. Previously Jinja2 templates ignored the newline symbols.
- Apply djlint formatting on the affected file

## QA

- Check out https://ubuntu-com-14094.demos.haus/certified/202307-31886
- Go to https://certification.canonical.com/certificates/2405-15681/
- Check that the "Notes" in demo shows line breaks where the C3 site does

## Issue / Card

Fixes #13870 and  [WD-11285](https://warthogs.atlassian.net/browse/WD-11285)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-11285]: https://warthogs.atlassian.net/browse/WD-11285?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
